pkgs/development/libraries/kde-frameworks/kpackage/allow-external-paths.patch

```diff
@@ -1,11 +1,11 @@
-Index: kpackage-5.30.0/src/kpackage/package.cpp
-===================================================================
---- kpackage-5.30.0.orig/src/kpackage/package.cpp
-+++ kpackage-5.30.0/src/kpackage/package.cpp
+diff --git a/src/kpackage/package.cpp b/src/kpackage/package.cpp
+index 5aec9fd..b15c933 100644
+--- a/src/kpackage/package.cpp
++++ b/src/kpackage/package.cpp
 @@ -820,7 +820,7 @@ PackagePrivate::PackagePrivate()
      : QSharedData(),
-       fallbackPackage(0),
-       metadata(0),
+       fallbackPackage(nullptr),
+       metadata(nullptr),
 -      externalPaths(false),
 +      externalPaths(true),
```

Comment:
@vandenoever I added this patch. It is required for Plasma to work at all. KDE upstream considers this to be a "security feature" but that's nonsense: if your "security" requires ignoring symlinks, you don't have "security".